docs/wiki/99_Developers/Code_Customization.md

```diff
@@ -0,0 +1,26 @@
+# Code customization
+## The project/ directory
+
+The `project/` directory is designed to house project-specific code and some data such as documents.  This includes instrument forms, instrument table schemas, and any customizations to other pages or modules in the Loris codebase.  
+The `project/` folder and its subdirectories are created by the install script, which also installs the `_config.xml_` file there.
+
+[[Instrument forms|Behavioural-Database]] (_.linst_ or php) should be stored under `project/instruments/` and their table schemas (.sql) should be stored under `project/tables_sql/` 
+
+## Backing up `project/`
+
+* Important: Commit your project/ directory and its contents to a separate private repo - e.g. on GitHub
+* Equally Important: Exclude all data and the _config.xml_ file (which contains a database credential) stored under project/ from commits/backups to your online repo (such as on GitHub).  Back up this data and the config file elsewhere on another server, using a cronjob. 
+
+## Tracking changes
+
+It is highly recommended to keep a Readme file under the project/ directory tracking all your project's customizations. This practice will help developers and also be useful at Release upgrade time, when merging any module updates in the codebase with your custom code.
+
+## Modifying and overriding code under the project/ directory
+
+Code stored under `project/modules/` and `project/libraries/` and `project/templates/` **will override** the file of the same name found in the same path under Loris root php/librariies/
```

Comment:
1. I think this should point out that using project/libraries is highly discouraged and likely to break on upgrades
2. `php/librariies/` is spelled wrong.

Reply:
@driusan Done!